docs/resources/mws_customer_managed_keys.md

```diff
@@ -16,38 +18,161 @@ Please follow this [complete runnable example](../guides/aws-workspace.md) with
 
 -> **Note** If you've used the resource before, please add `use_cases = ["MANAGED_SERVICES"]` to keep the previous behaviour.
 
+* Customer-managed key for managed services
```

Comment:
can you maybe do `###` instead? use https://registry.terraform.io/tools/doc-preview for preview

Reply:
That makes life a lot easier - I have updated the headings accordingly